### PR TITLE
added button to contribution guidelines and COC

### LINKED ISSUE #54: Add Instruction to take a look at Code of Conduct and Contribution Guidelines
Add Instruction to take a look at Code of Conduct and Contribution Guidelines on getmein web page, email sent to new member and README too.

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #54 

### Description

Added button for contribution guidelines and code of conduct.

###  Screenshot Before Change:
Screenshot (1387)

### Screenshot After Change:
Screenshot (1386)

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [x] Follows the code style of this project.
- [ ] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

deployment link : https://nlok5923-getmein-web-7.glitch.me/

### CODE REVIEW

**Comment by nlok5923** [comment]
@iiitv/project-maintainers  please review this pull request.

**Review by yashshah2820 — CHANGES_REQUESTED** [review]
@nlok5923 add 20px margin between the repo links and the submit button same as that between the span tag and the submit button.

**Comment by nlok5923** [comment]
After changes : 
Screenshot (1389)

**Review by yashshah2820 — APPROVED** [review]

File: views/index.html [hunk 1]
```diff
@@ -96,14 +96,18 @@
...
+              <a class="uk-text-center uk-button uk-button-default uk-border-pill uk-margin-small-top" target="_blank" href="https://github.com/iiitv/getmein-web/blob/master/CODE_OF_CONDUCT.md">CODE OF CONDUCT </a>
+            </div>
         </form>
       </div>
       <div class="message uk-text-center uk-text-success"></div>
     </main>
 
 
-    <footer class="uk-align-center uk-text-center">
+    <footer class="uk-align-center uk-text-center uk-margin-remove">
```
Comment (aashutoshrathi) [inline views/index.html:110]: Are we sure about removing this?
The margins don't look right. The page is also showing a disabled scrollbar and the footer content is being cut due to screen size.

Can we just calmly write some CSS to make sure content fits in 100vh always?

image

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]

**Comment by nlok5923** [comment]
@aashutoshrathi  Sir, there isn't much space for buttons while maintaining the same space as it is between the submit button and span element. that's why i has to apply `uk-margin-remove` to the footer div . but sir if we has to keep footer in the same way as before we has to go with this implementation while keeping the page fits in 100vh .


Screenshot (1401)

 or if we can make page scrollable we can go with buttons.
sir if there any other possible option please let me know.

**Pull request closed without merging**